Makefile

```diff
@@ -118,6 +118,23 @@ test-with-docker: docker-build
 		govuk/${CF_APP}:${GIT_COMMIT} \
 		make test
 
+.PHONY: build-with-docker
+build-with-docker: docker-build
+	docker run --rm \
+		-v "`pwd`:/var/project" \
+		-e COVERALLS_REPO_TOKEN=${COVERALLS_REPO_TOKEN} \
+		-e CIRCLECI=1 \
+		-e CI_BUILD_NUMBER=${BUILD_NUMBER} \
+		-e CI_BUILD_URL=${BUILD_URL} \
+		-e CI_NAME=${CI_NAME} \
+		-e CI_BRANCH=${GIT_BRANCH} \
+		-e CI_PULL_REQUEST=${CI_PULL_REQUEST} \
+		-e http_proxy="${http_proxy}" \
+		-e https_proxy="${https_proxy}" \
+		-e NO_PROXY="${NO_PROXY}" \
+		govuk/${CF_APP}:${GIT_COMMIT} \
+		npm set progress=false && npm install && npm rebuild node-sass && npm run build
```

Comment:
I think it would be tidier to put the commands in a file and then just call `bash script.sh`.

Reply:
Yeah good point. Updated and did a bit of tidying up of the Dockerfile as well.